### DISCUSSION TITLE
개발자 매뉴얼이 있나요?

### DISCUSSION THREAD

**Discussion by Sonak0930** [naverz/zepeto-studio-kor#949] [5. Q&A]
Zepeto Studio에서 제공하는 Guide를 보면서 따라하고 있는데, API manual이라던가 Zepeto에서 지원하는 feature들을 리스트로 모아놓은 게 있을까요? 기본적인 것들은 Unity와 비슷해 보이는데, 멀티플레이어같은 시스템은 달라서 질문드립니다.

++ Build it과 Unity로 만든 맵은 무슨 차이인가요? Unity에서 만드는 것과 차이가 있나요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. 현재 개발에 참고하실 만한 자료는 다음과 같습니다.
* 제페토 가이드 https://studio.zepeto.me/kr/guides/creating-your-own-world 
* 영상 가이드 
> 멀티플레이 : https://studio.zepeto.me/kr/guides/multiplay-tutorial
> 무작정 따라하기 : https://studio.zepeto.me/guides/world-tutorial-chapter1
* 공식 샘플 
> 스크립트 사용 샘플 : https://github.com/naverz/zepeto-script-sample
> 싱글플레이 샘플 : https://github.com/naverz/zepeto-world-sample
> 멀티플레이 샘플 : https://github.com/naverz/zepeto-multiplay-example
> 멀티플레이 애니메이션 모션 V2 동기화 샘플 : https://github.com/naverz/zepeto-studio-kor/discussions/835
> 다양한 뷰 샘플 : https://github.com/naverz/zepeto-studio-kor/discussions/826
> 무빙 블록 월드 샘플 : https://github.com/naverz/zepeto-studio-kor/discussions/902

API Document는 준비 중에 있습니다.

빌드잇은 제페토 월드를 만들기 위해 자체 제작한 PC용 맵 에디터입니다.
Unity 제페토 월드는 유니티 엔진을 사용하여 크리에이터가 보다 자유롭게 다양한 장르와 커스터마이즈된 UI 및
다양한 에셋을 활용한 월드를 개발할 수 있습니다. 

빌드잇과 유니티 월드의 가장 큰 차이는 스크립트의 사용 유무라고 할 수 있을 것 같습니다. 

감사합니다.
